Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/DebugControllers/GizmoLeft.prefab

```diff
@@ -917,7 +917,7 @@ Transform:
   - {fileID: 4391910669484542}
   m_Father: {fileID: 0}
   m_RootOrder: 0
-  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+  m_LocalEulerAnglesHint: {x: 90, y: 0, z: 0}
```

Comment:
How should these look with this rotation? The left one looks rotated forward and the right one looks rotated backward, but neither of them seem aligned with each other. Is that correct?

Reply:
With this rotation they are angled correctly in view.  Try it and see :D

Reply:
I tried it! My experience is described above with the rotations seeming off.

Reply:
OK, I'll reset the rotations and leave devs to align themselves, or you can update the controller models @davidkline-ms

Reply:
As checked in (with the rotation), this is what I'm seeing:

![image](https://user-images.githubusercontent.com/3580640/43349833-88975f7a-91b7-11e8-8155-d89032ced2cf.png)

The controllers are laying on the table, with the touchpad/thumbstick facing up and the tracking light ring facing forward. Is that what you expect?

Reply:
Rotation reset